helpers.php

```diff
@@ -389,3 +389,10 @@ function flux_path(string $path = ''): string
         return __DIR__ . ($path ? DIRECTORY_SEPARATOR . $path : $path);
     }
 }
+
+if (! function_exists('model')) {
+    function model(string $class): Illuminate\Database\Eloquent\Model
+    {
+        return app($class);
```

Comment:
This alone is not solving the issue. Is the intent to have something like a config file or anything?
Like 'user' => 'App\\Models\\User', and then every time the User model is needed, this function gets called with the parameter given from 'user' key? Or what is the Plan?